Bottom: 8f4cc7359f07ab2b13c49c1c3ad52bf0462d5011
Top:    dfaec038bcd007541876607d1686f8ef767e5055
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-04 00:31:07 +0000

Refresh of add-routines-to-render-just-a-

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 64005b0..92f2d32 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -618,16 +618,11 @@ static int assigned_bits = 0;
 void
 hidgl_fill_pcb_polygon (PolygonType *poly, const BoxType *clip_box, double scale)
 {
+  struct do_hole_info info;
   int stencil_bit;
   cairo_traps_t *traps;
-  struct do_hole_info info;
 
   info.scale = scale;
-
-//  bo_poly_to_traps (poly->Clipped);
-
-//  return;
-
   global_scale = scale;
 
   if (poly->Clipped == NULL)
